cohort-engine-distribution/src/main/bin/jvm.options

```diff
@@ -0,0 +1,17 @@
+#<!--  ************************************** -->
+#<!--  (C) Copyright IBM Corp. 2021, 2021     -->
+#<!--  SPDX-License-Identifier: Apache-2.0    -->
+#<!--  ************************************** -->
+# WebSphere Application Server Liberty Profile
+# This file contains options that are passed directly to the JVM on startup
+# See the InfoCenter for full documentation
+
+# Uncomment the following line to enable verbose output for class loading
+#-verbose:class
+
+# Uncomment the following line to enable debug messages during VM bootstrapping
+#-Dws.ext.debug
+
+-Xmx16G
```

Comment:
Was this intentionally 16GB? I'm not clear on how much we need, but that seems like an awful lot.

Reply:
Good catch. No we probably don't need this much. This was the value in the template from the service -example app I used as a template for these projects. I will lower it down.